Core/src/org/sleuthkit/autopsy/actions/DeleteFileBlackboardArtifactTagAction.java

```diff
@@ -95,7 +96,16 @@ protected void deleteTag(TagName tagName, BlackboardArtifactTag artifactTag, lon
 
             @Override
             protected Void doInBackground() throws Exception {
-                TagsManager tagsManager = Case.getCurrentCase().getServices().getTagsManager();
+                TagsManager tagsManager;
+                try {
+                    tagsManager = Case.getOpenCase().getServices().getTagsManager();
+                } catch (NoCurrentCaseException ex) {
+                    logger.log(Level.SEVERE, "Error untagging artifact. No open case found.", ex); //NON-NLS
+                    Platform.runLater(()
+                            -> new Alert(Alert.AlertType.ERROR, Bundle.DeleteFileBlackboardArtifactTagAction_deleteTag_alert(artifactId)).show()
+                    );
+                    return null;  
+                }
```

Comment:
There is an error in this code that we should fix, rather than repeat. The doInBackgroundMethod should just throw exceptions, rather than logging them and scheduling a task for the EDT to present the error to the user. This is why we have a done() method - to call get() to catch and handle exceptions thrown by doInBackground and to do things that need to be done in the EDT. Please eliminate the catches in doInBackground and put all of the exception handling in done(). Note that this may lead to unused imports to remove.